docs/user_guide/hopsworks/featurestore.rst

```diff
@@ -454,6 +454,68 @@ A feature store consists of five main components:
+.. figure:: ../../imgs/feature_store/secrets_manager.png
+    :alt: Hopsworks feature store secrets manager step 1
+    :target: `hopsworks_secrets_manager.png`_
+    :align: center
+    :scale: 20 %
+    :figclass: align-center
+
+As secret name enter *hopsworks/project/[MY_HOPSWORKS_PROJECT]/role/[MY_SAGEMAKER_ROLE]* replacing [MY_HOPSWORKS_PROJECT] with the name of the project hosting the Feature Store in Hopsworks and [MY_SAGEMAKER_ROLE] with the AWS role used by the SageMaker instance that should access the Feature Store. Select next twice and finally store the secret. Then click on the secret in the secrets list and take note of the *Secret ARN*.
+
+.. _hopsworks_secrets_manager2.png: ../../_images/secrets_manager2.png
+.. figure:: ../../imgs/feature_store/secrets_manager2.png
+    :alt: Hopsworks feature store secrets manager step 2
+    :target: `hopsworks_secrets_manager2.png`_
+    :align: center
+    :scale: 20 %
+    :figclass: align-center
+
+**Granting access to the secret to the SageMaker notebook role**
+
+In the AWS management console go to *IAM*, select *Roles* and then the role that is used when creating SageMaker notebook instances. Select *Add inline policy*. Choose *Secrets Manager* as service, expand the *Read* access level and check *GetSecretValue*. Expand Resources and select *Add ARN*. Paste the ARN of the secret created in the previous step. Click on *Review*, give the policy a name und click on *Create policy*.
+
+.. _hopsworks_aws_policy.png: ../../_images/aws_policy.png
+.. figure:: ../../imgs/feature_store/aws_policy.png
+    :alt: Hopsworks feature store set policy
+    :target: `hopsworks_aws_policy.png`_
+    :align: center
+    :scale: 20 %
+    :figclass: align-center
+
+**Installing hops-util-py and connecting to the Feature Store**
+To be able to access the Hopsworks Feature Store the hops-util-py library needs to be installed. One way of achieving this is opening a Python notebook in SageMaker and installing the latest hops-util-py together with a native depdendency using the following command. This requires root access on the notebook. Note that the library will not be persistet. For information around how to permanently install a library to Sagemaker see `Install External Libraries and Kernels in Notebook Instances <https://docs.aws.amazon.com/sagemaker/latest/dg/nbi-add-external.html>`_. ::
+
+    !sudo yum -y install cyrus-sasl-devel && pip install git+https://github.com/logicalclocks/hops-util-py.git
+
+You can now connect to the Feature Store::
+
+    import hops.featurestore as fs
+    fs.connect('my_instance.us-east-2.compute.amazonaws.com', 'my_project')
+
+If you have troubles connecting then ensure that the Security Group of your Hopsworks instance on AWS are configured to allow incoming traffic from your SageMaker instance. See `VPC Security Groups <https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html>`_. for more information.
```

Comment:
@steffengr `troubles` -> `trouble` and add a comma after `connecting`

Reply:
Done I should get a spell checker for my editor.